src/makeInfinityAdder.js

```diff
@@ -3,8 +3,24 @@
 /***
  * @return {function}
  */
-function makeInfinityAdder() {
-  // write code here
+function makeInfinityAdder(initialValue = 0) {
+  let sum = initialValue;
+
+  const makeAdder = (value) => {
```

Comment:
The `makeAdder` function should not accept a default parameter. The task specifies that there's no need to set a default value for the `adder` function argument.